### PR TITLE
Bug 1215926 - Fix strings in pin dialog r=albertopq

### PR DESCRIPTION
**Pull request by benfrancis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=68ff1051d74a22e922c1a308be6829b188a1666f

File: apps/system/js/pin_page_system_dialog.js [hunk 1]
```diff
@@ -128,7 +128,6 @@
     switch (action) {
       case 'pin':
         activeApp.appChrome.pinPage();
-        this._banner.show();
```
Comment (albertopq) [inline apps/system/js/pin_page_system_dialog.js]: We should probably remove the _banner in the dialog, and use the systemBanner in the appChrome for all the cases

File: apps/system/locales/system.en-US.properties [hunk 2]
```diff
@@ -647,11 +647,11 @@ pinning-unpin-type=Unpin "{{type}}"
 
 # LOCALIZATION NOTE (pinning-pin-page): This string is used as
 # panel title. "Pin" indicates the action to pin a page.
-pinning-pin-page=Pin Page
+pinning-pin-page=Pin page
 
 # LOCALIZATION NOTE (pinning-pin-site): This string is used as
 # panel title. "Pin" indicates the action to pin a site.
-pinning-pin-site=Pin Website
+pinning-pin-site=Pin site
```
Comment (albertopq) [inline apps/system/locales/system.en-US.properties]: The spec says 'Website', isn't it? https://mozilla.app.box.com/s/a25ot7ruu5u0mvxjirhrtu8d3rt5r107

File: apps/system/js/pin_page_system_dialog.js (hunk 1 above)
Comment (benfrancis) [inline apps/system/js/pin_page_system_dialog.js]: Yeah I tried to remove that and things broke but I couldn't figure out why. It looks like a lot of the code used by the door hanger hasn't been cleaned up when the door hanger was removed.

File: apps/system/locales/system.en-US.properties (hunk 2 above)
Comment (benfrancis) [inline apps/system/locales/system.en-US.properties]: Guillaume suggested to UX that it be changed to "Pin site" on Friday (for consistency) and Eric and Liz agreed.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b52c0143b17e042cf5c407e7b6999b6ff11c4431

**Pull request merged**